src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/client/ChipsRestClient.java

```diff
@@ -24,14 +26,14 @@
 
     @PostConstruct
     void init() {
-        chipsRestUrl = chipsRestHost + CHIPS_REST_ENDPOINT_URI_VAR;
+        chipsRestUrl = chipsRestHost + chipsRestEndpoint;
     }
 
-    public void sendToChips(ChipsKafkaMessage message) {
+    public void sendToChips(ChipsKafkaMessage message) throws RestClientException {
```

Comment:
RestClientException is a RuntimeException so doesn't need to be declared

Reply:
Removed declaration